.github/workflows/ci.yml

```
@@ -797,6 +797,10 @@ jobs:
         with:
           submodules: true
 
+      - name: capture git state in GitRev.hs
+        run: |
+          ./build.sh gitrev
```

Since this is a single shell command, we can simplify this to:

```suggestion
        run: ./build.sh gitrev
```

---

Fixed with the last force-push